lib/zendesk_api/resources.rb

```diff
@@ -277,83 +277,6 @@ class SatisfactionRating < ReadResource
     has Group
   end
 
-  class Search
-    class Result < Data; end
-
-    # Creates a search collection
-    def self.search(client, options = {})
-      unless (%w{query external_id} & options.keys.map(&:to_s)).any?
-        warn "you have not specified a query for this search"
-      end
-
-      ZendeskAPI::Collection.new(client, self, options)
-    end
-
-    # Quack like a Resource
-    # Creates the correct resource class from the result_type passed in
-    def self.new(client, attributes)
```

Comment:
This is confusing, usually, you would use `initialize`, but in this case, `.new` is a factory method for other classes.